tsdb/db.go

```diff
@@ -245,7 +238,28 @@ func newDBMetrics(db *DB, r prometheus.Registerer) *dbMetrics {
 		Name: "prometheus_tsdb_size_retentions_total",
 		Help: "The number of times that blocks were deleted because the maximum number of bytes was exceeded.",
 	})
-
+	m.minTime = prometheus.NewGaugeFunc(prometheus.GaugeOpts{
+		Name: "prometheus_tsdb_lowest_timestamp_seconds",
```

Comment:
You don't know the unit the tsdb is working with, these metrics are they way they are for good reasons.

Reply:
Hm.. Can we make TSDB unit-aware?